rust/src/operations/optimize.rs

```diff
@@ -576,54 +584,51 @@ impl MergePlan {
         mut self,
         object_store: ObjectStoreRef,
         snapshot: &DeltaTableState,
+        max_concurrent_tasks: usize,
+        #[allow(unused_variables)] // used behind a feature flag
+        max_spill_size: usize,
+        min_commit_interval: Option<Duration>,
```

Comment:
This is fine for now, but eventually we might want to put these execution settings in a struct.